Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -240,7 +240,12 @@ class HomepageViewController: UIViewController, HomePanel, GleanPlumbMessageMana
             presentedViewController?.dismiss(animated: false, completion: nil)
         }
 
-        // Adjust layout for rotation, cells needs to be relayout
+        // reload JumpBackIn section as the order of cells and their size can change depending on size class
+        if let index = viewModel.indexOfShownSection(.jumpBackIn) {
+            collectionView.reloadSections(IndexSet([index]))
+        }
+
+        // Adjust layout for rotation, cells need to re-layout
         collectionView.collectionViewLayout.invalidateLayout()
```

Comment:
Ah, I would've thought that `invalidateLayout()` would be enough there to trigger recalculation of the layout. At least it was enough before 😸